staking page crashes

After lunching godcr, going to the staking page cashes the app, sometimes you might need to switch between the staking page and other pages in quick successions before the crash occurs.

See the log below:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x10c49d2]

goroutine 998 [running]:
[github.com/planetdecred/dcrlibwallet.(*MultiWallet).ReloadVSPList(0xc0001d6380,](http://github.com/planetdecred/dcrlibwallet.(*MultiWallet).ReloadVSPList(0xc0001d6380,) 0x0, 0x0)
        /home/kennedy/Projects/pkg/mod/[github.com/planetdecred/dcrlibwallet@v1.6.2-0.20220315164532-e871b673497a/vsp.go:141](http://github.com/planetdecred/dcrlibwallet@v1.6.2-0.20220315164532-e871b673497a/vsp.go:141) +0x6b2
[github.com/planetdecred/godcr/ui/page/staking.(*Page).loadPageData.func1(0xc00052a400)](http://github.com/planetdecred/godcr/ui/page/staking.(*Page).loadPageData.func1(0xc00052a400))
        /home/kennedy/Projects/src/[github.com/dreacot/godcr/ui/page/staking/overview.go:141](http://github.com/dreacot/godcr/ui/page/staking/overview.go:141) +0x1d6
created by [github.com/planetdecred/godcr/ui/page/staking.(*Page).loadPageData](http://github.com/planetdecred/godcr/ui/page/staking.(*Page).loadPageData)
        /home/kennedy/Projects/src/[github.com/dreacot/godcr/ui/page/staking/overview.go:138](http://github.com/dreacot/godcr/ui/page/staking/overview.go:138) +0x3f
```

--- a/ui/page/staking/overview.go
+++ b/ui/page/staking/overview.go
@@ -138,7 +138,9 @@ func (pg *Page) loadPageData() {
 	go func() {
 		if len(pg.WL.MultiWallet.KnownVSPs()) == 0 {
 			// TODO: Does this page need this list?
-			pg.WL.MultiWallet.ReloadVSPList(pg.ctx)
+			if pg.ctx != nil {
+				pg.WL.MultiWallet.ReloadVSPList(pg.ctx)
+			}
 		}
 
 		totalRewards, err := pg.WL.MultiWallet.TotalStakingRewards()

